code/game/objects/effects/spawners/random/trash.dm

```diff
@@ -6,16 +6,23 @@
 /obj/effect/spawner/random/trash/garbage
 	name = "garbage spawner"
 	loot = list(
-		/obj/effect/spawner/random/trash/food_packaging = 56,
-		/obj/item/trash/can = 8,
-		/obj/item/shard = 8,
-		/obj/effect/spawner/random/trash/botanical_waste = 8,
-		/obj/effect/spawner/random/trash/cigbutt = 8,
-		/obj/item/reagent_containers/syringe = 5,
-		/obj/item/light/tube/broken = 3,
-		/obj/item/food/deadmouse = 2,
-		/obj/item/light/tube/broken = 1,
+		/obj/effect/spawner/random/trash/food_packaging = 20,
+		/obj/item/trash/can = 15,
+		/obj/item/shard = 10,
+		/obj/effect/spawner/random/trash/cigbutt = 10,
+		/obj/effect/spawner/random/trash/botanical_waste = 5,
+		/obj/item/reagent_containers/glass = 5,
+		/obj/item/broken_bottle = 5,
+		/obj/item/reagent_containers/glass/bowl = 5,
+		/obj/item/light/tube/broken = 5,
+		/obj/item/light/tube/broken = 5,
```

Comment:
```suggestion
		/obj/item/light/tube/broken = 10,

Reply:
I meant to make one the broken bulb and the other the broken lighttube.  Fixed with https://github.com/tgstation/tgstation/pull/64538/commits/82c2644da7f4bf273a6973df4317a04df16f148a